ios/ElectrodeApiImpl/APIImpls/Navigation/MiniAppNavViewController.swift

```diff
@@ -23,6 +23,7 @@ open class MiniAppNavViewController: UIViewController, ENNavigationProtocol {
     public var finish: Payload?
     public var delegate: ENNavigationDelegate?
     public var navigateWithRoute: NavigateWithRoute?
+    public var pushToExistingViewController: Bool = true
```

Comment:
adding this to a`MiniAppNavViewController.swift` limits the ability for the apps to create their own VCs with the delegates. See that there are also other methods added to the MiniAppVC. Let's have a discussion.

Reply:
This boolean value is required when pushing a miniappNavViewController on top of the existing navigationController. It is still okay if someone else wants to create their own VC and use ENCoreDelegate, ENCoreDelegate will check if the VC is a instance of MiniAppNavViewController, if VC is not a instance of MiniAppNavViewController, this boolean value is not being used.